lib/smart_answer_flows/benefit-cap-calculator.rb

```diff
@@ -200,12 +201,23 @@ def define
           option weekly_benefit_cap
         end
 
+        save_input_as :family_type
+
+        next_node do
+          question :enter_postcode?
+        end
+      end
+
+      #Q7 Future flow - Enter a postcode
+      postcode_question :enter_postcode? do
         calculate :benefit_cap do |response|
-          sprintf("%.2f", config.weekly_benefit_cap_amount(chosen_cap, response))
+          region = config.is_london?(response) ? :london : :national
```

Comment:
Minor: It seems odd to have a method which returns a truth-y or false-y value which we then immediately convert into symbols, especially as we're doing that conversion in two places. I think it'd be better if we had a method called something like `#region` on `config` which returned the appropriate symbol.